.github/workflows/test.yml

```
@@ -60,3 +60,10 @@
       - name: test with Gradle Wrapper
         run: ./gradlew test
 
+      - name: Generate coverage report
+        run: ./gradlew jacocoTestReport
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)